bin/metrics/main.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const glob = require('glob');
+const jsc = require('jscodeshift');
+const flowParser = require('jscodeshift/parser/flow');
+const tsxParser = require('jscodeshift/parser/tsx');
+const {Lambda} = require('@aws-sdk/client-lambda');
+
+/* eslint-disable no-console */
+
+const lambda = new Lambda({
+  region: 'eu-west-1',
+});
+
+exports.main = async function (commitID, commitDate) {
```

Comment:
I should be able to call cli locally so it might be needed to decouple it from lambda call

Reply:
I rewritten cli with `meow` and now you can display help message:

```
$ sg-metrics --help
```

To be able to get metrics but use it locally CLI now have flag `--dry` which logs result to `stdout`